.storybook/main.js

```diff
@@ -1,16 +1,22 @@
 const path = require('path');
 
 module.exports = {
-  "stories": [
+  stories: [
     "../src/**/*.stories.mdx",
     "../src/**/*.stories.@(js|jsx|ts|tsx)"
   ],
-  "addons": [
-    "@storybook/addon-links",
-    "@storybook/addon-essentials",
-    "@storybook/addon-a11y",
-    "@storybook/addon-knobs",
-    "@storybook/addon-storysource"
+  addons: [
+    {
+      name: "@storybook/addon-essentials",
+      options: {
+        actions: false,
+        backgrounds: true,
+        controls: true,
+        docs: false,
+        viewport: true,
+        toolbars: false
+      }
+    }
```

Comment:
The Essentials addon contains some of the previous, separate addons.

I removed the duplicate addons and the ones that weren't compatible (caused npm or build errors).